src/msg/async/AsyncConnection.cc

```diff
@@ -2364,12 +2358,19 @@ ssize_t AsyncConnection::write_message(Message *m, bufferlist& bl)
                              << " data=" << header.data_len
                              << " off " << header.data_off << dendl;
 
-  complete_bl.claim_append(bl);
+  if (bl.length() <= 256) {
```

Comment:
Hmmm... Maybe I should add additional condition, so single, small bufferptr is not incorporated into outcoming_bl.